server/sonar-web/src/main/js/helpers/handlebars/dashboardUrl.js

```diff
@@ -18,7 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 module.exports = function (componentKey, componentQualifier) {
-  let url = '/dashboard/index?id=' + encodeURIComponent(componentKey);
+  let url = window.baseUrl + '/dashboard/index?id=' + encodeURIComponent(componentKey);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)